Form/DataTransformer/LegacyModelsToArrayTransformer.php

```diff
@@ -17,11 +17,9 @@
 use Symfony\Component\Form\Exception\UnexpectedTypeException;
 
 /**
- * Class ModelsToArrayTransformer.
+ * This class should be used with Symfony <2.7 only and will be deprecated with 3.0.
```

Comment:
Not sure. If people are stuck with 2.3 and have no alternative to using this class, they should not get a deprecation notice.